tardis/conftest.py

```diff
@@ -35,6 +42,54 @@ def pytest_addoption(parser):
     parser.addoption("--slow-test-data", dest="slow-test-data",
```

Comment:
I thought we changed this to integration test (at least @yeganer and I discussed this).